apps/swirl-docs/src/lib/navigation/src/data/apiDocs.data.ts

```diff
@@ -2,14 +2,207 @@ import { NavItem } from "../navigation.model";
 
```

Comment:
**question:** Can this be added to the gitignore? Seems to be generated and contains paths to your local file system that wouldn't work on other machines anyway.

Reply:
good catch!